src/flows/shared/steps/OrganizeGallery/CollectionDnd.tsx

```diff
@@ -59,15 +59,16 @@ function CollectionDnd({ collections }: Props) {
   }, []);
 
   const activeCollection = useMemo(() => {
-    return collectionOrder.find(({ id }) => id === activeId);
-  }, [activeId, collectionOrder]);
+    return collections.find(({ id }) => id === activeId);
```

Comment:
So activeCollection now checks the collection from SWR instead of the local state's collection?

Reply:
yup exactly- this will handle the small edge case where the shadow of the collection being dragged may be stale (e.g. old collection name)